controllers/gslb_controller_test.go

```diff
@@ -1127,12 +1127,12 @@ func reconcileAndUpdateGslb(t *testing.T, s testSettings) {
 		if res != (reconcile.Result{RequeueAfter: time.Second * 30}) {
 			t.Error("reconcile did not return Result with Requeue")
 		}
+		err = s.reconciler.Get(context.TODO(), s.request.NamespacedName, s.gslb)
+		if err != nil {
+			t.Fatalf("Failed to get expected gslb: (%v)", err)
```

Comment:
just curious, can't we just use require.NoError here ? Fatal breaks testrun, is it expected ?

Reply:
@kuritka that is something i just moved from one place to another, so it is the old part of the test suite. I will try to enhance it, good points